src/xcov-core/IDE/IDESchemeActionCodeCoverageFile.h

```diff
@@ -8,8 +8,8 @@
 
 @interface IDESchemeActionCodeCoverageFile : DVTCoverageDataContainer
 
-@property (nonatomic, copy) NSArray *lines;
-@property (nonatomic, copy) NSArray *functions;
-@property (nonatomic, copy) NSString *documentLocation;
+@property(nonatomic, copy) NSArray *lines;
```

Comment:
Sure, the only reason I did this is to keep the style consistent, but I didn't pay enough attention to what was the initial style. 😞 
I'll revert these changes and (in another PR) I will adapt my code in order to be consistent with the initial style. 😉